crates/network/src/any/mod.rs

```diff
@@ -73,5 +73,7 @@ impl Network for AnyNetwork {
 
     type ReceiptResponse = AnyTransactionReceipt;
 
-    type HeaderResponse = WithOtherFields<Header>;
+    type HeaderResponse = Header;
```

Comment:
should this remain `WithOtherFields<Header>` ?

Reply:
Header is getting flattened into block in RPC, so if block has additional fields those will get consumed by header's `others`. There's no concept of body/header division in RPC, so we can't really tell which field belongs to a header or to a body of the block

Reply:
right, that makes sense